README.md

```diff
@@ -1 +1,27 @@
-# CFReT_data
\ No newline at end of file
+# CFReT Project
+
+## Data
+
+The data used in this project is a [Cell Painting assay](https://www.moleculardevices.com/applications/cell-imaging/cell-painting#gref) on [cardiac fibroblasts](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5588900/#:~:text=Definition%20by%20function,%2C%20and%20glycoproteins5%2C6.) from 3 patients that suffered from cardiac arrest. 
+
+In this Cell Painting, there are five channels:
+
+- `d0` (Nuclei)
+- `d1` (Endoplasmic Reticulum)
+- `d2` (Golgi/Plasma Membrane)
+- `d3` (Mitochondria)
+- `d4` (F-actin)
+
+![Composite_Figure.png](example_figs/Composite_Figure.png)
+
+## Goal
+
+The goal of this project is to identify a biomarker associated with cardiac fibroblasts that indicates if the scarring caused by cardiac arrest is reversed.
```

Comment:
```suggestion
The goals of this project are to identify morphology features that distinguish cardiac patients and to discover a cell morphology biomarker associated with drug treatment to reverse fibrosis scarring caused by cardiac arrest.
```

I _think_ this is more accurate :)

Reply:
Mine was definitely the shortened version haha. I believe this has more detail and is what we had discussed with the collaborators.